### PR TITLE
Document that booked invoices can not be updated

### PR DESCRIPTION
**Pull request by steftrenson**
We've got feedback that it wasn't clear that booked invoices can not be updated. Let's improve the documentation for the endpoint.

### CODE REVIEW

**Review by lorgan3 — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -2914,7 +2914,7 @@ Draft a new invoice.
 
 ### invoices.update [POST /invoices.update]
 
-Update an invoice.
+Update a draft invoice. Booked invoices can not be updated.
```
Comment (MathiasDL) [inline apiary.apib]:
```suggestion
Update a draft invoice. Booked invoices cannot be updated.
```

File: src/05-invoicing/invoices.apib [hunk 2]
```diff
@@ -305,7 +305,7 @@ Draft a new invoice.
 
 ### invoices.update [POST /invoices.update]
 
-Update an invoice.
+Update a draft invoice. Booked invoices can not be updated.
```
Comment (MathiasDL) [inline src/05-invoicing/invoices.apib]:
```suggestion
Update a draft invoice. Booked invoices cannot be updated.
```

**Review by MathiasDL — CHANGES_REQUESTED** [review]

**Comment by MathiasDL** [comment]
As seen all over the codebase, "can not" doesn't exist in English, it is "cannot"

**Comment by steftrenson** [comment]
I can not believe that

**Review by MathiasDL — APPROVED** [review]

**Review by lorgan3 — APPROVED** [review]

**Review by mrtus — APPROVED** [review]

**Pull request merged**